typescript/src/api-doc-types/ConstExportDoc.ts

```diff
@@ -5,25 +5,12 @@ import { ModuleDoc } from './ModuleDoc';
 
 export class ConstExportDoc extends ExportDoc {
   docType = 'const';
-  variableDeclaration = this.declaration as VariableDeclaration;
```

Comment:
This is technically a breaking change. We use it for a workaround, but I just wanted to let you know.

Not a big deal from our side though 😄

Reply:
I'll revert this removal then. No need to add unnecessary breaking changes.